src/main/webapp/components/BuildSnapshotContainer.js

```diff
@@ -8,11 +8,25 @@ import {Wrapper} from "./Wrapper";
  * @param {*} props input property containing an array of build data to be
  * rendered through BuildSnapshot.
  */
-export const BuildSnapshotContainer = (props) => {
-  const data = props.data;
-  return (
-    <Wrapper>
-      {data.map(snapshotData => <BuildSnapshot data={snapshotData}/>)}
-    </Wrapper>
-  );
-}
+export const BuildSnapshotContainer = React.memo((props) => {
+    const SOURCE = '/data';
+    const [data, setData] = React.useState([]);
+
+    /**
+    * Fetch data when component is mounted.
```

Comment:
nit: please add a space before the `*` here and below to align the column of stars in the comment with the first start from the `/**` line.

Reply:
Updated alignment here 48edcfe